### PR TITLE
[1LP][RFR] Ignore providers based on use_for_sprout

### PR DESCRIPTION
**Pull request by john-dupuy**
Purpose or Intent
=================

Only add providers which have `use_for_sprout = True` to the sprout DB.

Also removing a redundant setting.

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 7dc99b89e965b482c6bd4663b57073a28a7dd43f:

`sprout/appliances/tasks.py`:
- :bangbang: Line 413:11: **E261** *at least two spaces before inline comment*
- :bangbang: Line 515:11: **E261** *at least two spaces before inline comment*
- :bangbang: Line 518:12: **E111** *indentation is not a multiple of four*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 03f0a7a434f3f1210f0cbd8d3afe187db37b38f4:

`sprout/appliances/tasks.py`:
- :bangbang: Line 413:11: **E261** *at least two spaces before inline comment*
- :bangbang: Line 413:12: **E262** *inline comment should start with '# '*
- :bangbang: Line 515:11: **E261** *at least two spaces before inline comment*
- :bangbang: Line 515:12: **E262** *inline comment should start with '# '*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

File: sprout/sprout/settings.py [hunk 1]
```diff
@@ -132,7 +132,6 @@
 CELERY_RESULT_SERIALIZER = 'pickle'
 CELERY_TIMEZONE = "UTC"
 CELERY_DISABLE_RATE_LIMITS = True
-CELERY_TIMEZONE = 'UTC'
```
Comment (digitronik) [inline sprout/sprout/settings.py]: :+1:

File: sprout/appliances/tasks.py [hunk 2]
```diff
@@ -408,13 +408,16 @@ def poke_trackerbot(self):
             if build_date <= (parsetime.today() - timedelta(days=group.template_obsolete_days)):
                 # It is already obsolete, so ignore it
                 continue
-        provider, create = Provider.objects.get_or_create(id=template["provider"]["key"])
+        if conf.cfme_data.management_systems.get(template["provider"]["key"], {}).get(
```
Comment (mshriver) [inline sprout/appliances/tasks.py]: Not related to your change ... but holy crap this method is too long :)

**Pull request merged**